packages/connect-wallets/src/lib/nova-wallet/index.ts

```diff
@@ -0,0 +1,14 @@
+import { BaseDotsamaWallet } from '../base-dotsama-wallet';
+import logo from './NovaWalletLogo.svg';
+
+export class NovaWallet extends BaseDotsamaWallet {
+  extensionName = 'polkadot-js';
```

Comment:
Hi @valentunn, please update `extensionName` to ensure dapps can differentiate between Polkadot Js and Nova